src/shared-components/optionButton/index.tsx

```diff
@@ -115,43 +118,44 @@ export const OptionButton = ({
 /**
  * A presentational component to match the display of an OptionButton with an icon
  */
-export const OptionButtonNotClickable = ({
-  borderRadius = DEFAULT_BORDER_RADIUS,
-  icon,
-  optionType,
-  subtext,
-  text,
-  ...rest
-}: OptionButtonNotClickableProps) => (
-  <Style.DisplayContainer
-    borderRadius={borderRadius}
-    containerType="none"
-    // eslint-disable-next-line react/jsx-props-no-spreading
-    {...rest}
-  >
-    <OptionButtonContent
-      // The buttonType does not matter for this component
-      buttonType="primary"
-      icon={icon}
-      optionType={optionType}
-      selected={false}
-      subtext={subtext}
-      text={text}
-    />
-  </Style.DisplayContainer>
-);
+export const OptionButtonNotClickable: React.FC<OptionButtonNotClickableProps> =
+  ({
+    borderRadius = DEFAULT_BORDER_RADIUS,
+    icon,
+    optionType,
+    subtext,
+    text,
+    ...rest
+  }) => (
+    <Style.DisplayContainer
+      borderRadius={borderRadius}
+      containerType="none"
+      // eslint-disable-next-line react/jsx-props-no-spreading
+      {...rest}
+    >
+      <OptionButtonContent
+        // The buttonType does not matter for this component
+        buttonType="primary"
+        icon={icon}
+        optionType={optionType}
+        selected={false}
+        subtext={subtext}
+        text={text}
+      />
+    </Style.DisplayContainer>
+  );
 
 /**
  * Similar OptionButton styling without click elements
  */
 OptionButton.NotClickable = OptionButtonNotClickable;
 
 OptionButton.propTypes = {
-  borderRadius: PropTypes.string,
+  borderRadius: BORDER_RADIUS_PROP_TYPES,
   buttonType: PropTypes.oneOf(['primary', 'secondary']),
   icon: PropTypes.node,
   onClick: PropTypes.func.isRequired,
-  optionType: PropTypes.oneOf(['radio', 'checkbox']).isRequired,
+  optionType: PropTypes.oneOf(['radio', 'checkbox'] as const).isRequired,
```

Comment:
It is also unclear to me why `buttonType` does not get flagged by `optionType` does, if `as const` directive is not used. 🤷